google/cloud/storage/notification.py

```diff
@@ -233,7 +233,7 @@ def _set_properties(self, response):
         self._properties.clear()
         self._properties.update(response)
 
-    def create(self, client=None, timeout=_DEFAULT_TIMEOUT):
+    def create(self, client=None, timeout=_DEFAULT_TIMEOUT, retry=DEFAULT_RETRY):
```

Comment:
Not idempotent, must have no retry argument or retry=None default.

Reply:
7c1f93dde1765bfb2744660498485649ca2153b7